### ISSUE TITLE
Inconsistent sizes for FPUDataPointer

### ISSUE THREAD

**Issue by mlgiraud** [NationalSecurityAgency/ghidra#6856]
The x86 specification specifies the FPUDataPointer register as either size 4 or size 8, depending on whether we use the 32 bit or 64 bit spec. See here: https://github.com/NationalSecurityAgency/ghidra/blob/2eb39242f9c2c37df796c66b2830f9aa48bfb034/Ghidra/Processors/x86/data/languages/ia.sinc#L86
Later on however, in the `FSAVE` constructor, the size is fixed to 4 bytes: https://github.com/NationalSecurityAgency/ghidra/blob/2eb39242f9c2c37df796c66b2830f9aa48bfb034/Ghidra/Processors/x86/data/languages/ia.sinc#L4783
This doesn't seem correct to me. Also shouldn't the sleigh compiler raise an error on parsing this? The left hand side is specified to be 4 bytes, but, if we are in the 64 bit case, the right hand side has size 8 (the varnode is defined with size 8 in Line 86).
Am i misunderstanding something, or is this an actual issue?

**Comment by GhidorahRex** [comment]
Yes, the pcode for these instructions is implicitly truncating the FDP/FIP. Only 32-bits is ever used for each, even though in 64-bit mode they are 64-bits in width.

This doesn't appear to be an issue.
